backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  unitAccessibilityPriorityTypes UnitAccessibilityPriorityTypes? @relation(fields: [priorityTypeId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listings                       listings?                       @relation(fields: [listingId], references: [id], onDelete: Cascade)
+  unitRentTypes                  UnitRentTypes?                  @relation(fields: [unitRentTypeId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  status                         UnitsStatusEnum                 @default(unknown)
+
+  @@map("units")
+}
+
+model UnitsSummary {
+  id                             String                          @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  monthlyRentMin                 Int?                            @map("monthly_rent_min")
+  monthlyRentMax                 Int?                            @map("monthly_rent_max")
+  monthlyRentAsPercentOfIncome   Decimal?                        @map("monthly_rent_as_percent_of_income") @db.Decimal(8, 2)
+  amiPercentage                  Int?                            @map("ami_percentage")
+  minimumIncomeMin               String?                         @map("minimum_income_min")
+  minimumIncomeMax               String?                         @map("minimum_income_max")
+  maxOccupancy                   Int?                            @map("max_occupancy")
+  minOccupancy                   Int?                            @map("min_occupancy")
+  floorMin                       Int?                            @map("floor_min")
+  floorMax                       Int?                            @map("floor_max")
+  sqFeetMin                      Decimal?                        @map("sq_feet_min") @db.Decimal(8, 2)
+  sqFeetMax                      Decimal?                        @map("sq_feet_max") @db.Decimal(8, 2)
+  totalCount                     Int?                            @map("total_count")
+  totalAvailable                 Int?                            @map("total_available")
+  unitTypeId                     String?                         @map("unit_type_id") @db.Uuid
+  listingId                      String?                         @map("listing_id") @db.Uuid
+  priorityTypeId                 String?                         @map("priority_type_id") @db.Uuid
+  unitTypes                      UnitTypes?                      @relation(fields: [unitTypeId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  unitAccessibilityPriorityTypes UnitAccessibilityPriorityTypes? @relation(fields: [priorityTypeId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listings                       listings?                       @relation(fields: [listingId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+
+  @@map("units_summary")
+}
+
+// Note: [mfa_code] formerly max length 16; [first_name, middle_name, last_name] formerly max length 64
+// Note: [email] formerly lower case enforced
+model UserAccounts {
```

Comment:
This was previously called just "User"

Reply:
gosh I WISH!

`@Entity({ name: "user_accounts" })` we named it user_accounts in the db 

probably because I think users is a reserved word in postgres (or at least it can be in other dbs)

Reply:
Oh I was looking at the name of the class, which is `User`, my b!